source/paged-geom/BatchedGeometry.cpp

```diff
@@ -200,7 +200,7 @@ uint32 CountUsedVertices(IndexData *id, std::map<uint32, uint32> &ibmap)
          break;
 
       default:
-         throw new Ogre::Exception(0, "Unknown index buffer type", "Converter.cpp::CountVertices");
+         throw new Ogre::Exception(Exception::ERR_INVALIDPARAMS, "Unknown index buffer type", "Converter.cpp::CountVertices");

```

Comment:
I don't know if `0` and `1` used to be used for a specific purpose of just as "whatever just throw an exception", but given that `0` seems to be `ERR_CANNOT_WRITE_TO_FILE` (https://ogrecave.github.io/ogre/api/1.11/class_ogre_1_1_exception.html#a9bdf9b8a0a83b95818d676848beaa29b) and thus not really relevant, I tried to find suitable exception types for each error.